duckduckgo/duckduckgo-privacy-extension
# Don't overinject scripts into frames

**Reviewer:** @Charlie-belmer @kdzwinel 



## Description:

We were injecting into all frames on every onCommitted event, which was leading to injecting multiple instances into each frame. This PR fixes that.

## Steps to test this PR:
1. Install extension
2. Navigate to onet.pl, check console. There shouldn't be any errors related to content scripts


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
